src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int value = new Random().nextInt(100);
```

Comment:
Why are you using the number 100 here?
To answer this question to someone who will read your code, you can make this value constant.